docs/central-troubleshooting.rst

```diff
@@ -46,21 +46,21 @@ You may run into a "Could not connect with Server" error message when previewing
 
 To resolve this problem, first identify your upstream DNS servers. Run ``cat /run/systemd/resolve/resolv.conf`` to see your current list of nameservers with their IP addresses. They will look like this:
 
-  .. code-block:: console
+.. code-block:: console
```

Comment:
Above in this file, it says you can run `docker logs --tail 100 central_service_1` but the docker containers have hyphens now `docker logs --tail 100 central-service-1`

Reply:
Ah yes!